tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13598,6 +13606,78 @@ INLINE void static add_property_struct(char *func_name, int nreductions,
 }
 #endif
 
+void 
+add_vscale() 
+{
+  char token[60];
+  sprintf(token, "%d,%d", flg.min, flg.max);
+  print_token(" \"vscale-range\"=\"");
+  print_token(token);
+  print_token("\"");
+}
+
+bool
+is_vscale_feat(char *mod)
+{
+  bool result = false;
+  int len = sizeof(*CPU_VSCALE)-1;
+
+  if(mod == NULL) {
+    return result;
+  }
+  
+  for(int i=0; i < len; ++i) {
+    if(strcmp(mod, CPU_VSCALE[i]) == 0) {
+      /* TODO: check if there is other argument for sve scale range to use */
+      result = true;
+      /* if NOT SVE ARG */
+      flg.min = 1;
+      flg.max = 16;
+      /* else */
+      //flg.min = argument1;
+      //flg.max = argument2;
+      break;
+    }
+  }
+
+  return result;
+}
+
+bool
```

Comment:
There's something wrong with this function, it spilled mod equals to `+sve2,+sve` which resulted in `is_vscale_feat()` couldn't recognize `+sve2` in simple `strcmp()`.